apps/authentication/migrations/0042_auto_20190918_1958.py

```diff
@@ -0,0 +1,18 @@
+# Generated by Django 2.1.11 on 2019-09-18 17:58
+
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('authentication', '0041_auto_20190911_2238'),
+    ]
+
+    operations = [
+        migrations.AlterField(
+            model_name='onlineuser',
+            name='allergies',
+            field=models.TextField(blank=True, null=True, verbose_name='matallergier/preferanser'),
+        ),
```

Comment:
Also, I don't think this migration is relevant to the PR, so I would suggest removing it.

Reply:
Done.